lib/iris/experimental/regrid.py

```diff
@@ -491,6 +491,31 @@ def _regrid_area_weighted_array(
         cached_x_bounds.append(x_bounds)
         cached_x_indices.append(x_indices)
 
+    # Move y_dim and x_dim to last dimensions
+    x_dim_orig = copy.copy(x_dim)
+    y_dim_orig = copy.copy(y_dim)
```

Comment:
Stephen queried the use of `copy.copy` and I agree it isn't required here. I'll add this comment to #3596.